src/utils/credentials.ts

```diff
@@ -23,24 +23,24 @@ export function askForCookies() {
       {
         name: "domain",
         message:
-          "What is your Retool domain? (e.g. my-org.retool.com). Don't include https:// or http://",
+          "What is your Retool domain? (e.g., my-org.retool.com). Don't include https:// or http://",
         type: "input",
       },
       {
         name: "xsrf",
         message:
-          "Great, now what is your XSRF token? (e.g. 26725f72-8129-47f7-835a-bca0e5dbcfe6) \n  You can find this by logging into Retool, and opening up the cookies inspector in your browser's dev tools.\n  In Chrome, hit ⌘+⌥+I (Mac) or Ctrl+Shift+I (Windows, Linux) to open the dev tools.\n  Application tab > your-org.retool.com from the Cookies section on the left > double click the cookie value and copy it.",
+          "What is your XSRF token? (e.g., 26725f72-8129-47f7-835a-bca0e5dbcfe6) \n  You can find this by logging into Retool, and opening the cookies inspector in your browser's dev tools.\n  In Chrome, hit ⌘+⌥+I (Mac) or Ctrl+Shift+I (Windows, Linux) to open the dev tools.\n  Application tab > your-org.retool.com from the Cookies section on the left > double click the cookie value and copy it.",
         type: "input",
       },
       {
         name: "accessToken",
-        message: `Last thing! What is your access token? It's a very long string, also found in the cookies inspector.`,
+        message: `What is your access token? It's a very long string, also found in the cookies inspector.`,
         type: "input",
       },
     ])
     .then(function (answer: Credentials) {
       persistCredentials(answer);
-      console.log("Credentials saved/updated successfully!");
+      console.log("Credentials saved and updated successfully!");
```

Comment:
If "saved" and "updated" are interchangeable we can just pick one and go with it. I wasn't sure so I included both, but if they mean the same thing we can go with just one.

Reply:
Originally there was a difference between save and update, but I don't think it's worth highlighting to user. 

We should just go with "saved" everywhere.